homeassistant/components/smarttub/config_flow.py

```diff
@@ -24,6 +25,12 @@ class SmartTubConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
 
+    def __init__(self) -> None:
+        """Instantiate config flow."""
+        super().__init__()
+        self._reauth_input = None
```

Comment:
```suggestion
        self._reauth_input = {}
```

Reply:
Why? This will cause the `_reauth_input is None` conditional to be true

Reply:
Needed for `default=self._reauth_input.get(CONF_EMAIL)` or you can keep it `None` and check that `_reauth_input` is a dict

Reply:
But if you are keeping it in the other step not needed at all